### DISCUSSION TITLE
Extract Method refactoring with ValueTuple return instead of out parameters

### DISCUSSION THREAD

**Discussion by VolodymyrBS** [dotnet/roslyn#53022] [Q&A] [answered]
Hi!
Not sure if it was not asked before but is it possible to change Visual Studio Extract Method refactoring behaviour to extract method and return local value tuple instead of creating multiple out parameters?

**Comment by ryzngard** [comment, marked as answer]
@VolodymyrBS I don't think this has been asked for before. Feel free to file an issue as a feature request
